apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java

```diff
@@ -43,9 +45,19 @@ protected void onCreate(Bundle savedInstanceState) {
         sSharedPreferences = getSharedPreferences("Sasquatch", Context.MODE_PRIVATE);
         StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder().detectDiskReads().detectDiskWrites().build());
 
-        String serverUrl = sSharedPreferences.getString(SERVER_URL_KEY, null);
-        if (serverUrl != null) {
-            MobileCenter.setServerUrl(serverUrl);
+        String logUrl = sSharedPreferences.getString(LOG_URL_KEY, null);
+        if (logUrl != null) {
+            try {
+                Method setLogUrl;
+                try {
+                    setLogUrl = MobileCenter.class.getMethod("setLogUrl", String.class);
+                } catch (NoSuchMethodException e) {
+                    setLogUrl = MobileCenter.class.getMethod("setServerUrl", String.class);
```

Comment:
I know this is a test application but why don't you put a comment why you are doing it here. I know why and you know why but others don't know. 😄